docs/how-to-guides/sparksql-source-notes.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: default
+title: Storing secrets in Azure Key Vault
+parent: How-to Guides
+---
+
```

Comment:
Tables in Blob storages can also be used as this 'SparkSQLSrouce' (when using synapse as spark provider). Shall we include this case here?